packages/cli/src/ErrorReporting.ts

```diff
@@ -32,6 +33,17 @@ export const initErrorHandling = async () => {
 		},
 	});
 
+	const seenErrors = new Set<string>();
+	addGlobalEventProcessor((event, { originalException }) => {
+		if (originalException instanceof NodeError && originalException.severity === 'warning')
+			return null;
+		if (!event.exception) return null;
+		const eventHash = createHash('sha1').update(JSON.stringify(event.exception)).digest('base64');
+		if (seenErrors.has(eventHash)) return null;
+		seenErrors.add(eventHash);
```

Comment:
This seems to have some potential for a memory leak, doesn't it? We're never cleaning this up. Maybe we should consider a cleanup routine? As instances can run for months this would grow endlessly and possibly even lead to poor performance.

Reply:
Considering we're only storing hashes and even a user with 1000 unique errors, that'd be less than 50kb memory.